plugins/modules/backup_plan_info.py

```diff
@@ -20,7 +20,7 @@
   backup_plan_names:
     type: list
     elements: str
-    required: true
+    required: false
```

Comment:
nitpick: `false` is the default value for `required`. So this line can be avoided.